cmake/targets/BuildLibArchive.cmake

```diff
@@ -4,7 +4,7 @@ register_repository(
   REPOSITORY
     libarchive/libarchive
   COMMIT
-    898dc8319355b7e985f68a9819f182aaed61b53a
+    7ce42547f682ea79f9382d580bd97355c0885a0f
```

Comment:
this commit doesn't exist

Reply:
seems like the github api is weird

https://api.github.com/repos/libarchive/libarchive/git/ref/tags/v3.7.7
https://github.com/libarchive/libarchive/releases/tag/v3.7.7